start-site/src/main/resources/application.yml

```diff
@@ -86,19 +86,19 @@ initializr:
         mappings:
           - compatibilityRange: "[2.2.0.RELEASE,2.4.0.M1)"
             version: Hoxton.SR12
-          - compatibilityRange: "[2.4.0.M1,2.4.0.M1]"
-            version: 2020.0.0-M3
-            repositories: spring-milestones
-          - compatibilityRange: "[2.4.0.M2,2.4.0-M3]"
-            version: 2020.0.0-M4
-            repositories: spring-milestones
           - compatibilityRange: "[2.4.0.M4,2.4.0]"
             version: 2020.0.0
           - compatibilityRange: "[2.4.1,2.5.x-SNAPSHOT)"
             version: 2020.0.3
           - compatibilityRange: "2.5.x-SNAPSHOT"
             version: 2020.0.4-SNAPSHOT
             repositories: spring-snapshots,spring-milestones
+          - compatibilityRange: "[2.6.0.M1,2.6.0.M1]"
```

Comment:
Compatibility ranges must cover a supported generation. An easier way to write this would be to limit the range to `2.6.0-SNAPSHOT` so that the next entry covers the remaining of `2.6.x`.  `"[2.6.0.M1,2.6.x-SNAPSHOT)"` should do it

Reply:
Thanks